tests/tests/test-multisigs/test-multisigs.ts

```diff
@@ -0,0 +1,118 @@
+import "@polkadot/api-augment";
+import "@moonbeam-network/api-augment";
+import { blake2AsHex, createKeyMulti } from "@polkadot/util-crypto";
+import { u8aToHex } from "@polkadot/util";
+import { expect } from "chai";
+import { describeDevMoonbeam } from "../../util/setup-dev-tests";
+import {
+  alith,
+  ALITH_ADDRESS,
+  DOROTHY_ADDRESS,
+  CHARLETH_ADDRESS,
+  BALTATHAR_ADDRESS,
+  baltathar,
+} from "../../util/accounts";
+
+describeDevMoonbeam("Multisigs - perform multisigs operations", (context) => {
+  const threshold = 2;
+  let otherSignatories = [BALTATHAR_ADDRESS, CHARLETH_ADDRESS];
+
+  let call: any;
+  let encodedCall: any;
+  let encodedCallHash: any;
+
+  // create multisig accountId
+  let encodedMultisigId = createKeyMulti([ALITH_ADDRESS, BALTATHAR_ADDRESS, CHARLETH_ADDRESS], 2);
+  let multisigId = u8aToHex(encodedMultisigId.slice(0, 20));
+  let multisigInfo: any;
+
+  it("Should create a multisig operation with asMulti", async function () {
+    // encode and hash the call we want to dispatch as a multisig operation
+    call = context.polkadotApi.tx.balances.transferKeepAlive(DOROTHY_ADDRESS, 20);
+    encodedCall = call.method.toHex();
+    encodedCallHash = blake2AsHex(encodedCall);
+
+    const block = await context.createBlock(
+      context.polkadotApi.tx.multisig
+        .asMulti(threshold, otherSignatories, null, encodedCall, {})
+        .signAsync(alith)
+    );
+
+    // take the info of the new multisig operation saved in storage
+    multisigInfo = await context.polkadotApi.query.multisig.multisigs(multisigId, encodedCallHash);
```

Comment:
Is this storing the value from one test case to be used in another? If yes, this should be refactored to be done in the setup phase with `before`